src/Components/WebAssembly/Sdk/src/Microsoft.NET.Sdk.BlazorWebAssembly.csproj

```diff
@@ -14,6 +14,8 @@
     <ExcludeFromSourceBuild>false</ExcludeFromSourceBuild>
 
     <IsShipping>false</IsShipping>
+    <!-- Workaround while there is no 6.0 SDK available, suppress unsupported version error -->
+    <NETCoreAppMaximumVersion>6.0</NETCoreAppMaximumVersion>
```

Comment:
Unfortunately this and similar changes won't improve anything. The errors about `.net6.0` come from attempts to build test assets, not the product or test projects.

This project already has the property set. See https://github.com/dotnet/aspnetcore/pull/26788/files#diff-1ea18ff65faa2ae6fed570b83747086d0317f5e4bc325064f6c14319a9c4ff67R40

Reply:
> The template tests (Blazor and regular) need the <NETCoreAppMaximumVersion>6.0</NETCoreAppMaximumVersion> hack

Okay, so what should I do then? I'm a big confused 😄

Reply:
To impact the test-time builds, likely need to change TemplateTests.props.in and the code that uses those templates. Probably also need something specific to the Helix test environment that ensures `$(NETCoreAppMaximumVersion)` is set e.g. something in the Directory.Build.props file(s) used on the Helix agents.